.github/workflows/mosip-nexus-chart-lint-publish.yml

```
@@ -0,0 +1,61 @@
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths:
+      - 'MosipNexus/helm/**'
+  workflow_dispatch:
+    inputs:
+      IGNORE_CHARTS:
+        description: 'Provide list of charts to be ignored separated by pipe(|)'
+        required: false
+        default: ''
+        type: string
+      CHART_PUBLISH:
```

actionlint (1.7.12)

[error] "options" can not be set to "chart_publish" input because its input type is not "choice"

(events)